spotify_player/src/client/mod.rs

```diff
@@ -199,7 +199,9 @@ impl Client {
 
                 state.player.write().mute_state = new_mute_state;
             }
-            PlayerRequest::StartPlayback(p) => {
+            PlayerRequest::StartPlayback(p, random) => {
+                println!("random: {} ", random);
```

Comment:
is this for debugging? If yes, can you remove it?

Reply:
yep, and removed!